fix userId not being send as customer_id in track & page events

ruemic opened the pull request:
@ndhoule `customer_id` in our integration for track and page events is currently always returning null.  This fixes the issue. LMK if anything.

ruemic commented:
@ndhoule can this get merged please?

sperand-io commented:
hey @ruemic thanks for the PR, sorry for the delay

 comments inline — if you can't get to them today lmk and i'll make the changes myself. thanks agaiN!

sperand-io commented on lib/index.js:
```diff
@@ -5,6 +6,7 @@
 
 var integration = require('analytics.js-integration');
 var foldl = require('foldl');
+var analytics = require('analytics.js-core');
```
no need to require that here — `analytics` is available on the `integration` object, accessible within methods w/ 

```
this.analytics.user().id()
this.analytics.user().anonymousId();
//etc
```

sperand-io commented on lib/index.js:
```diff
@@ -58,8 +60,8 @@ Blueshift.prototype.page = function(page) {
-  properties.customer_id = page.userId();
-  properties.anonymousId = page.anonymousId();
+  properties.customer_id = analytics.user().id();
+  properties.anonymousId = analytics.user().anonymousId();
```
`this.analytics.user().id()`

ruemic commented:
Cool, I will update right now and let u know

> On Aug 11, 2015, at 3:38 PM, Chris Sperandio notifications@github.com wrote:
> 
> hey @ruemic https://github.com/ruemic thanks for the PR, sorry for the delay
> 
> comments inline — if you can't get to them today lmk and i'll make the changes myself. thanks agaiN!
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/segment-integrations/analytics.js-integration-blueshift/pull/1#issuecomment-130099547.

sperand-io commented:
thanks man!

ruemic commented:
@sperand-io updated to use analytics from integration object

ruemic commented:
ok, updated the PR, squashed into existing commit

> On Aug 11, 2015, at 3:41 PM, Chris Sperandio notifications@github.com wrote:
> 
> thanks man!
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/segment-integrations/analytics.js-integration-blueshift/pull/1#issuecomment-130100104.
